File: translation_output/w_test/mistral/codenet/java/python/p00290.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
20 2

Expected output:
40


```

import java.util.Scanner;

public class p00290 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int d = scanner.nextInt();
        int c = scanner.nextInt();
        System.out.println(d*c);

    }
}


```
[/INST]
```python
d = int(input())
c = int(input())
print(d*c)
```</s>

File: Results/DER/Translation/translation_output/w_test/granite/codenet/python/java/p00290.java
System:
You are an expert java programmer and assistant

Question:
Translate the following code from python to java and enclose your solution inside ```java```.
A sample test case is provided below:

Test input:
20 2

Expected output:
40


```
a,b= map(int,input().split())

print(a*b)

```


Answer:
```java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = scanner.nextInt();
        int b = scanner.nextInt();
        System.out.println(a * b);
    }
}
```

Test input:
```
20 2
```

Expected output:
```
40
```<|endoftext|>
